plugins/modules/aci_ip_sla_monitoring_policy.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_ip_sla_monitoring_policy
+short_description: Manage IP SLA Monitoring Policies (fv:IPSLAMonitoringPol)
+description:
+- Manage IP SLA Monitoring Policies used for L4-L7 Policy Based Redirection
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  sla_policy:
+    description:
+    - SLA Policy name
+    type: str
+  sla_type:
+    description:
+    - Type of monitoring
+    type: str
+    choices: [ icmp, tcp, l2ping ]
+  sla_port:
+    description:
+    - Port to monitor for TCP SLAs
+    - If converting an existing TCP SLA to ICMP or L2ping, set sla_port to 0
+    type: int
+  frequency:
+    description:
+    - How often to probe
+    type: int
+  multiplier:
+    description:
+    - How many probes must fail for the SLA to be down
+    type: int
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
```

Comment:
add annotation and owner documentation fragments

Reply:
added